src/__tests__/fakes/button.ts

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
`fakes/`? how about `__fixtures__/` adjacent to the `__tests__/` folder

Reply:
needed a name 😄